demo.sh

```diff
@@ -0,0 +1,39 @@
+printf  "%s\n" "----------------------1-------------------------"
+printf  "%s\n" "Find movies wich are shows that day and around this time(+/-1hour):"
+curl -X GET 'http://localhost:8080/client/find-screenings?date=2021-03-01&startTime=15:00:00'
+
+printf  "%s\n" "----------------------2-------------------------"
+printf  "%s\n" "Get info about chosen screening: "
+curl -X GET 'http://localhost:8080/client/get-screening?date=2021-03-01&time=15:00:00&title=Harry%20Potter'
+
+printf  "%s\n" "----------------------3-------------------------"
+printf  "%s\n" "Reserve tickets:"
+curl -X POST 'http://localhost:8080/client/create-tickets/C3,C4/ADULT,STUDENT?date=2021-03-01&time=15:00:00&title=Harry%20Potter&firstName=Roman&secondName=Czerski'
```

Comment:
czy tak powinno wyglądać restowe api?

Reply:
Jeśli chodzi o to, że przesyłam dane w parametrach i nie używam JSON'a to ciężko mi znaleźć wytłumaczenie czemu tak to zrobiłem.